src/modules/UI/scenes/TransactionList/reducer.js

```diff
@@ -1,13 +1,23 @@
 // @flow
 
 import type { EdgeTransaction } from 'edge-core-js'
-import { combineReducers } from 'redux'
+import { type Reducer, combineReducers } from 'redux'
 
+import type { GuiContact, TransactionListTx } from '../../../../types.js'
 import type { Action } from '../../../ReduxTypes.js'
 
-export type TransactionsState = Array<EdgeTransaction>
+export type TransactionListState = {
+  +contactsList: Array<GuiContact>, // MISSING!!!
```

Comment:
Missing reducer.

Reply:
It appears that this reducer has been moved to a top-level reducer just called `contacts`